### ISSUE TITLE
#27 Defining ParsedInfo

### ISSUE THREAD

**Issue by jin8** [AutoDo/AutoDo#28]
```
api = {"class1":{"method1":"description","method2":"description", "method3":"description"},
       "class2":{"method1":"description"}}

readme = {"Introduction":"needuser input" , 
          "Requirements":"information need to execute",
          "Installation": "how to install--- need user input"
          "API Reference": "api link",
          "Dependency graph": "image of dependency graph",
          "Contributor & Maintainers":"get recent participant of the projects",
          "License": "get license of the project"}

graph = ?
```

This is the form expected to receive from the parser. I am currently work on graph data structure. So if there is update in graph data structure. I will leave comment to this issue as soon as possible.

**Comment by JunoJunho** [comment]
I checked the information format.

**Comment by jin8** [comment]
```
graph= [("class A","class B","method A to B"),
          ("class B","class C","method B to C"),
          ("class C","class A","method C to A"),
          ("class C", "class A", "method C' to A'"),
          ("class C", "class A", "method C'' to A''"),
          ("class A","class A","method A to A"),
          ("class B","class B","method B to B"),
          ("class C","class C","method C to C")]

```

graph data is list of tuple of source , destination, and method call.

**Comment by JunoJunho** [comment]
I completed the parser implementation

This is the output format

`[('Math(object)', 'Triangle', 'check_anglecheck_angle()'), ('Math(object)', 'Square', 'get_areaget_area()'), ('Math(object)', 'Square', 'get_heightget_height()'), ('Math(object)', 'Square', 'get_widthget_width()'), ('Math(object)', 'Circle', 'get_areaget_area()'), ('Math(object)', 'Circle', 'get_circumferenceget_circumference()')]`

**Comment by JunoJunho** [comment]
Check out the pull-request.

**Comment by JunoJunho** [comment]
This issue is resolved by pull-request #32
